cuedb/schema.go

```diff
@@ -110,48 +111,46 @@ func GetSchemaV1Metadata(schema cue.Value) (SchemaV1Metadata, error) {
 
 // V1_MODEL is the cue representation of
 // Model metadata
+type ModelV1Metadata struct {
+	Plural              string
+	SupportedExtensions []string
+}
+
 const V1_MODEL = `
```

Comment:
golint is going to complain about ALL_CAPS. Go style is CamelCase even for constants.